tests/color_refinement.py

```diff
@@ -50,7 +51,10 @@ def unit_test(write_csv_any=False, write_stdout_passed=True, write_stdout_fail=T
         for i in range(0, len(graphs) - 1):
             for j in range(i + 1, len(graphs)):
                 # Graph test
-                
+
+                fix_degrees(graphs[i])
```

Comment:
Zijn al deze fix_degrees nodig om twin removal te doen? Is het niet mogelijk om dat alleen daar te gaan doen?

Reply:
Doordat ik de preprocessing los heb gehaald van het bepalen van isomorphismen en automorphismen weet ik op dat tweede punt niet of er wel of niet twins waren verwijderd. Dus wat ik nu doe is in de preprocessing de degrees vastzetten op de property fix_degrees en de initial coloring altijd op die property te baseren. Maar daardoor moet je dus ook in de tests waar kleuringen voorkomen die fix_degrees() een keer runnen, omdat anders de property degree_fixed niet bestaat